ios/MaasCore/Sources/MaasComponents/Components/Button.swift

```diff
@@ -1,75 +1,160 @@
 public struct Button: View, Swappable {
-
+    
     public struct InputType {
         public let text: String
-        public let foreground: Color?
-        public let background: Color?
+        public let icon: Image?
+        public let foregroundColor: Color?
+        public let backgroundColor: Color?
+        public let isSmall: Bool
+        public let isHugging: Bool
```

Comment:
🤗 is this a term used in standard SwiftUI components? 🙂

Reply:
It's a term in UIKit 🙈 `contentHuggingPriority`